.github/workflows/ci.yml

```diff
@@ -38,6 +38,9 @@ jobs:
           python -m pip install --upgrade pip
           # setuptools needed for 3.12+ because of https://github.com/mesonbuild/meson/issues/7702.
           python -m pip install pytest setuptools
+
+          # symlink /bin/true to /usr/bin/getuto (or do we want to grab the script from github?)
+          sudo ln -s /bin/true /usr/bin/getuto
```

Comment:
Another possible alternative would be to set PORTAGE_TRUST_HELPER in the environment.

Reply:
Yes. 
In either case, my logic here was that getuto has its own tests (thanks sam); here we test portage and want to avoid spurious failures.

Reply:
Right. I'm just saying that instead of a symlink, we could set PORTAGE_TRUST_HELPER=true to turn it into a noop.

If the symlink is easier to manage, that's fine.